langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -320,6 +428,14 @@ isolated function getReplacementString(Groups groups, Replacement replacement) r
 # between matches, or after the last match.  If there are no matches, then
 # `[str]` will be returned.
 #
+# ```ballerina
+# string:RegExp r = re `,`;
+#
+# r.split("abc,cde,efg") ⇒ ["abc","cde","efg"]
+#
+# r.split("Not Valid") ⇒ ["Not Valid"]
```

Comment:
Shouldn't we use some other string here? It isn't quite an invalid scenario?

Reply:
[e2120f5](https://github.com/ballerina-platform/ballerina-lang/pull/39096/commits/e2120f595dd1902d93841371a8a3c871b3f7dde8)